### ISSUE TITLE
ORCID

### ISSUE THREAD

**Issue by maelle** [r-lib/desc#63]
One can now add ORCID for authors of a package on CRAN and it's shown with a nice icon in the CRAN page of a package e.g. https://cran.r-project.org/web/packages/fasterize/index.html

I'm wondering how to use `desc` to add an ORCID for an author. Should this be a separate argument of authors helpers, although the ORCID information is supposed to live in comment (as per https://cran.r-project.org/web/packages/submission_checklist.html "For persons with an ORCID identifier (see https://orcid.org/ for more information, provide the identifier via an element named "ORCID" in the comment argument of person(). Example: person("Achim", "Zeileis", comment = c(ORCID = "0000-0003-0918-3766")). ")?

Currently when trying to add the ORCID via comment I used this workflow

```r
descr <- desc::desc(find.package("desc"))
descr$add_author(given = "Nice",
                 family = "Developer",
                 comment = c(ORCID = "orcid_number"))
descr$write("DESCRIPTION")
```

Maybe this is not the right way to save DESCRIPTION? If it is, when using this in the file comment becomes "comment = structure("orcid_number", .Names = "ORCID"))" which I'm not sure would be parsed by CRAN (but I haven't tried).

**Comment by gaborcsardi** [comment]
>  Should this be a separate argument of authors helpers

yes

> Maybe this is not the right way to save DESCRIPTION?

No, it is  OK. You can also just do 

```r
desc_add_author(given = "Nice",
                 family = "Developer",
                 comment = c(ORCID = "orcid_number"))
```
to do it in one go.

> which I'm not sure would be parsed by CRAN (but I haven't tried).

It is fine for CRAN, but I agree that it does not look nice. Do you want to submit a PR to add  an `orcid` argument?

**Comment by maelle** [comment]
I wonder whether I should also add an `add_orcid` function/method, because I know I'd use it to complement existing DESCRIPTION files.

**Comment by llrs** [comment]
I think it would be useful to have `add_orcid`, it will help a lot with existing packages (maybe it will get used in usethis)

**Comment by maelle** [comment]
Fixed in #70 💃
